main/src/main/resources/org/clulab/numeric/HOLIDAY.tsv

```diff
@@ -0,0 +1,49 @@
+#
+# List of holidays in the United States extracted from the Jollyday api.
+# This list can be obtained with the following snippet:
+#
+#   import de.jollyday.{HolidayManager, HolidayCalendar, ManagerParameters}
+#   import collection.JavaConverters._
+#   val holidayManager = HolidayManager.getInstance(ManagerParameters.create(HolidayCalendar.UNITED_STATES))
+#   holidayManager.getCalendarHierarchy.getChildren.keySet.asScala.
+#                   map(holidayManager.getHolidays(2021, _).asScala).flatten.map(_.getDescription)
+#
+#
+Lincoln's Birthday
+Easter Monday
+Washington's Birthday
+Labour day
+César Chávez's Birthday
+Seward's Day
+Christmas Eve
+Mardi Gras
+Battle of Bennington
+Arbor Day
+West Virginia Day
+Election day
```

Comment:
I'm not sure of the capitalization throughout the list, some Day and some day.  Will we match OK either way?

Reply:
The lexicon is added as case insensitive and everything is lowercased in HolidayNormalizer, so this should not be a problem. Maybe, I should lowercase this file to avoid any confusion.